gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -448,6 +449,33 @@ local key_params = {
 	walkkey = Key("+walk", "WALK")
 }
 
+-- handle looking at doors
+function HUDDrawTargetIDDoors(tData)
+	local client = LocalPlayer()
+	local ent = tData:GetEntity()
+
+	if not IsValid(client) or not client:IsTerror() or not client:Alive()
```

Comment:
If using a player metatable based function on an entity, you should check whether this ent is a player first.

Reply:
I do not understand what you mean, as the client (the entity we are talking about?) is retrieved using the `LocalPlayer()` function, thus i do not think an additional check for a player entity is needed. Or i might not understand your comment and you would need to further explain this.